universal_notifications/backends/push/gcm.py

```diff
@@ -7,16 +7,18 @@
 from django.core.exceptions import ImproperlyConfigured
 from push_notifications import NotificationError
 from push_notifications.settings import PUSH_NOTIFICATIONS_SETTINGS as SETTINGS
+
 from universal_notifications.backends.push.utils import get_app_settings
 
 try:
-    from urllib.request import Request, urlopen
     from urllib.parse import urlencode
+    from urllib.request import Request, urlopen
 except ImportError:
     # Python 2 support
-    from urllib2 import Request, urlopen
     from urllib import urlencode
 
+    from urllib2 import Request, urlopen
```

Comment:
why do we keep support for python 2?

Reply:
No idea. I was not doing cleanup old versions. We can do it later.